src/utils/functions/meal.ts

```diff
@@ -0,0 +1,75 @@
+import { PetType } from '@/utils/constants/db'
+
+// Define activity ratios for dogs based on age, neutered status, and activity
+export const getDogActivityRatio = (
```

Comment:
Can throw this and the cat func into chat gpt ask it refactor and remove the nesting. I feel can just use 'if' statement everything, don't need 'else if' cos all are returns